snippets/elixir.snippets

```diff
@@ -197,3 +197,15 @@ snippet >i pipe to inspect
 	|> IO.inspect
 snippet >il pipe to inspect with label
 	|> IO.inspect(label: "${1:label}")
+snippet gs basic genserver structure
```

Comment:
This is going to be used infrequently, so could we give it a longer trigger please? :)

Reply:
Sure - in my personal snippets it was `genserver`, so I'll change it to that